Source/Core/Core/HW/Memmap.cpp

```diff
@@ -189,6 +248,34 @@ static u32 GetFlags()
 
 void Init()
 {
+  const auto GetMEM1Size = []() -> u32 {
```

Comment:
Our naming convention is to use snake_case for variables, even if they're lambda functions.

```suggestion
  const auto get_mem1_size = []() -> u32 {
```

(also I'm not sure explicitly specifying the return type is necessary?)

Reply:
If lambdas are to use snake_case, this should be stated in the Contributing.md document.